src/component/Header.jsx

```diff
@@ -1,13 +1,40 @@
 import { useContext } from "react";
 import { UserContext } from "../context/UserContext";
-
+import { useEffect, useState } from "react";
+import { getTopics } from "../api"
+import { Link } from "react-router-dom";
 
 const Header = () => {
-const {user} = useContext(UserContext)
+  const { user } = useContext(UserContext);
+  const [topics, setTopics] = useState([])
+
+useEffect(() => {
+getTopics().then((data) => {
+    setTopics(data)
+})
+}, [])
 
-    return <div className="title"><h1>NewsBox</h1>
-    <p className="login-statement">Logged in as: {user}</p>
-    </div>
-}
+  return (
+    
+    <body>
```

Comment:
This is causing a dom error in your terminal because you've made a main section in your app.jsx. You cannot have the body of a html page be a child of a main section.